src/styles/_catalog.scss

```
@@ -1,22 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: 400px;
+  grid-auto-rows: 300px;
+  place-content: center;
+  grid-gap: 48px;
+  justify-content: center;
+  justify-items: center;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
```

move to the variable
```suggestion
    grid-template-columns: repeat(2, $cardWidth);
```